install/infra/modules/k3s/local.tf

```diff
@@ -0,0 +1,8 @@
+locals {
```

Comment:
**opinion(non-blocking):** This is purely my opinion and deviates from the standards, but I personally like having local variables declared closer to the place where they're being used - if I can see all the context in a single pane in my editor then I need to do less chasing across a module. Purely subjective opinion on my part, no action needed.